### PR TITLE
libmnl: new package at 1.4.0

### PR DESCRIPTION
**Pull request by sebastianertz**

### CODE REVIEW

**Comment by dylanaraps** [comment]
Care to write a little description as to what this library is and what it is exactly for? (The package itself LGTM)

File: public/libmnl/version [hunk 1]
```diff
@@ -0,0 +1 @@
+171f89699f286a5854b72b91d06e8f8e3683064c5901fb09d954a9ab6f551f81  libmnl-1.0.4.tar.bz2
```
Comment (dylanaraps) [inline public/libmnl/version]: 1.0.4 1

**Comment by sebastianertz** [comment]
> Care to write a little description as to what this library is and what it is exactly for? (The package itself LGTM)
wireguard need libmnl

**Comment by dylanaraps** [comment]
I think it's an optional dependency (unless you specifically need the functionality from the library)?

https://openwrt.org/packages/pkgdata/wireguard

Either way, LGTM :+1:

**Comment by konimex** [comment]
Please make a separate PR for the main wireguard package. After that we can merge both at the same time if it's "good enough".

**Comment by sebastianertz** [comment]
> I think it's an optional dependency (unless you specifically need the functionality from the library)?
> 
> https://openwrt.org/packages/pkgdata/wireguard
> 
> Either way, LGTM +1

see: https://openwrt.org/packages/pkgdata/wireguard-tools

**Comment by konimex** [comment]
I meant, you don't have to close this PR. Just make a separate branch for wireguard.

**Pull request closed without merging**